// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

message t_phr_info { 
    required uint32 serv_cell_id = 1;
    required uint32 ph_min = 2;
    required uint32 ph_max = 3;
    required uint32 p_cmax_min = 4;
    required uint32 p_cmax_max = 5;
}

// For explanation, see include/srsran/mac/phr_report.h
message mac_sched_ul_phr_indication {
    required uint64 timestamp = 1;
    required uint32 ue_index = 2;
    required uint32 rnti = 3;
    repeated t_phr_info ph_reports = 4;
}





